extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/OidcRequestContextProperties.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.oidc.common;
+
+import java.util.Map;
+
+public class OidcRequestContextProperties {
+
+    private Map<String, Object> properties;
```

Comment:
```suggestion
    private final Map<String, Object> properties;
```

it can be easily changed without breaking anything if later you find it shouldn't be (which I don't think it should)